app/src/main/java/com/duckduckgo/app/bookmarks/migration/AppDatabaseBookmarksMigrationCallback.kt

```diff
@@ -85,6 +85,7 @@ class AppDatabaseBookmarksMigrationCallback(
                     val entity = Entity(UUID.randomUUID().toString(), it.title, it.url, BOOKMARK)
                     entitiesMigration.add(entity)
                     favouriteMigration.add(Relation(folderId = SavedSitesNames.FAVORITES_ROOT, entityId = entity.entityId))
+                    favouriteMigration.add(Relation(folderId = SavedSitesNames.BOOKMARKS_ROOT, entityId = entity.entityId))
```

Comment:
ok, so this will fix the bug we had in the migration.

But, if I'm not wrong, all users that had favorites and are already migrated, they will have favorites that are not bookmarks, and that's not addressed here. Can you create a task in Asana, and define how important is to migrate them?

Reply:
sounds good! added it https://app.asana.com/0/0/1204733967627383/f